holidays/calendars/mongolian.py

```
@@ -0,0 +1,235 @@
+    }
+
+    def _get_holiday(self, holiday: str, year: int) -> tuple[Optional[date], bool]:
+        estimated_dates = getattr(self, f"{holiday}_DATES", {})
+        exact_dates = getattr(self, f"{holiday}_DATES_{_CustomCalendar.CUSTOM_ATTR_POSTFIX}", {})
+        dt = exact_dates.get(year, estimated_dates.get(year, ()))
+        return date(year, *dt) if dt else None, year not in exact_dates
+
+    def _get_holiday_set(self, holiday: str, year: int) -> Iterable[tuple[date, bool]]:
+        estimated_dates = getattr(self, f"{holiday}_DATES", {})
+        exact_dates = getattr(self, f"{holiday}_DATES_{_CustomCalendar.CUSTOM_ATTR_POSTFIX}", {})
+        for year in (year - 1, year):
```

Pylint (3.3.7)

[refactor] Redefining argument with the local name 'year'

(R1704)